Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -1102,12 +1127,15 @@ protected void IncreaseDimensionIndex()
 
             if (currentIndex != dimensionIndex)
             {
-                FilterThemesByDimensions();
+                SetupThemes();
                 forceUpdate = true;
             }
         }
 
-        public void ForceUpdateThemes()
```

Comment:
For a less disruptive change, instead of removing this method, would it make sense to mark it as Obsolete and redirect this to the new method?